webrtc-stats.html

```diff
@@ -1093,7 +1100,7 @@
               </dt>
               <dd>
                 <p>
-                  Represents the <code>track.id</code> property.
+                  Represents the <code>track.id</code> property of the track.
```

Comment:
Syntactically better. We have a pretty heavy collision of "id" names here, but I guess it's clear enough.